lib/routes/guozaoke/index.ts

```diff
@@ -0,0 +1,102 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseRelativeDate } from '@/utils/parse-date';
+import { config } from '@/config';
+import cache from '@/utils/cache';
+import asyncPool from 'tiny-async-pool';
+
+export const route: Route = {
+    path: '/default',
+    categories: ['bbs'],
+    example: '/guozaoke/default',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '过早客',
+    maintainers: ['xiaoshame'],
+    handler,
+    url: 'guozaoke.com/',
+};
+
+async function handler() {
+    const url = `https://www.guozaoke.com/`;
+    const res = await got({
+        method: 'get',
+        url,
+        headers: {
+            Cookie: config.guozaoke.cookies,
+            'User-Agent': config.ua,
+        },
+    });
+    const $ = load(res.data);
+
+    const list = $('div.topic-item').toArray();
+    const maxItems = 20; // 最多取20个数据
+    const items = [];
+    for (const item of list) {
+        if (items.length >= maxItems) {
+            break;
+        }
+        const $item = $(item);
+        const title = $item.find('h3.title a').text();
+        const link = $item.find('h3.title a').attr('href');
```

Comment:
Please remove the URL hash from `link` **HERE** otherwise the same item will be shown again in RSS reader when there's a new reply **AND** causes RSSHub visiting the same page with the same content again **AND** storing duplicated content in cache.